pubsub/pulsar/pulsar.go

```diff
@@ -202,23 +225,32 @@ func (p *Pulsar) Publish(ctx context.Context, req *pubsub.PublishRequest) error
 	)
 	topic := p.formatTopic(req.Topic)
 	producer, ok := p.cache.Get(topic)
+
+	schemaMetadata, hasSchema := p.metadata.topicSchemas[req.Topic]
```

Comment:
rename this variable as it conflicts with the global type definition

Reply:
Done